sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_models.hpp

```diff
@@ -1168,6 +1168,10 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
     ~CertificateOperationProperties() = default;
   };
 
+  /**
+   * @brief Represents a deleted certificate in a KeyVault
```

Comment:
Consistently end with a period.
```suggestion
   * @brief Represents a deleted certificate in a KeyVault.
```